sys/arm64/arm64/trap.c

```diff
@@ -427,6 +427,16 @@ data_abort(struct thread *td, struct trapframe *frame, uint64_t esr,
 		panic("data abort in critical section or under mutex");
 	}
 
+#if __has_feature(capabilities)
+	if ((esr & ISS_DATA_DFSC_MASK) == ISS_DATA_DFSC_LC_SC &&
```

Comment:
The commit log here should say "restore SEGV_LOADTAG" reporting

Reply:
I've squashed it into the commit it fixes while leaving the qemu workaround separate so we can find and potentially revert it later.